lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
                     "transition_to": "pending_review",
+                    "attributes": {
+                        "presentation_sequence": 1
+                    },
                     "notifications": [
                         {
                             "notification_type": "email",
                             "name": "Sufia::Workflow::PendingReviewNotification",
                             "to": ["approving"],
                             "cc": []
                         }
+                    ],
+                    "methods": [
+                        "CurationConcerns::Workflow::DeactivateObject"
                     ]
-                },
-                {
+                }, {
+                    "name": "depositor_review",
+                    "from_states": [{"names": ["complete", "pending_review"], "roles": ["depositor_reviewing"]}],
+                    "transition_to": "depositor_review",
```

Comment:
I'd like to see this change to `changes_required` to reflect that this is a state that a person has requested. (Told you I was picky about names -- hope I didn't disappoint!)